### PR TITLE
[Cherry 2.6.x] Flaky tests mitigation

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

File: tests/tests/test_demo_artifact.py [hunk 1]
```diff
@@ -94,6 +95,7 @@ def run_demo_script_up(exit_cond=exit_cond):
 
     # Give the test a timeframe, as the script might run forever,
     # if something goes awry, or the script is not brought down properly.
+    @flaky(max_runs=3)  # https://tracker.mender.io/browse/MEN-4495
```
Comment (lluiscampos) [inline tests/tests/test_demo_artifact.py:98]: This test started failing after traefik migration (iow post Mender 2.6).

I think we should not mark it `flaky` here

Comment (oleorhagen) [reply tests/tests/test_demo_artifact.py:98]: I see, sure. Due to my somewhat stupid commit setup for this though, I think I'll actually add an extra commit in which I remove it from `test_demo` :-1:

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Review by lluiscampos — APPROVED** [review]
You might need to adjust the copyright year for test_mtls. And kind of funny that we ended up bumping the copyright hear for test_demo_artifact but not actually changing the code :)

Thanks for porting it!

**Comment by oleorhagen** [comment]
> You might need to adjust the copyright year for test_mtls.

Seemed to pass just fine :thinking: Should I be worried? @lluiscampos 

> And kind of funny that we ended up bumping the copyright hear for test_demo_artifact but not actually changing the code :)

Lol, yes!

If none of the above is a problem, I will go ahead and merge, there was no errors with `flaky` itself.

**Comment by lluiscampos** [comment]
> > You might need to adjust the copyright year for test_mtls.
> 
> Seemed to pass just fine thinking Should I be worried? @lluiscampos
> 

It seems like the copyright check in CI has not been ported to 2.6, you got lucky :P

> 
> If none of the above is a problem, I will go ahead and merge, there was no errors with `flaky` itself.

 :green_circle:

**Pull request merged**